src/OctoshiftCLI.Tests/Services/ReclaimServiceTests.cs

```diff
@@ -745,7 +745,7 @@ public async Task ReclaimMannequin_AlreadyMapped_Force_Reclaim()
         }
 
         [Fact]
-        public async Task ReclaimMannequin_FailtToReclaim_LogsError_Throws_OctoshiftCliException()
+        public async Task ReclaimMannequin_FailedToReclaim_LogsError_Throws_OctoshiftCliException()
```

Comment:
```diff
-Failt
+Failed
```

It's possible this meant to be 
```suggestion
        public async Task ReclaimMannequin_FaultToReclaim_LogsError_Throws_OctoshiftCliException()
```

But that didn't seem likely from context...